core-addon/content-script.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Send an evet to the web page.
+ *
+ * @param {Object} message
+ *        A JSON-serializable object to send back to the
+ *        page. It must have the following structure:
+ *        {type: "message-type", data: {...}}
+ */
+function respondToPage(message) {
+  console.debug(`Ion: sending response ${data.type} to page`);
+  const data = message.data || {};
+  let event = new CustomEvent(data.type, { detail: data });
+  window.dispatchEvent(data);
+}
+
+// Listen for messages from the page.
+// If the message was from the page script, show an alert.
+window.addEventListener("web-check", event => {
+  if (event.source === window) {
+    console.debug("Ion - 'web-check' message received from the page");
+    // TODO: use webextension-polyfill for chrome compatibility
```

Comment:
@hamilton is there any way for a single rollup configuration to generate two different artefacts?

We need to generate a `background.js` (which we do) and also to generate a `content-script.js`. We want to go through Rollup with this as we need to include the webextension-polyfill library.

Reply:
Rollup can create multiple bundles from one config. To achieve this, you [can export default an array of configs](https://rollupjs.org/guide/en/#configuration-files). Look in this section for a number of options.

Reply:
Thanks! That worked!